features/support/world.coffee

```diff
@@ -5,6 +6,13 @@ class World
 
   mychaPath: path.join __dirname, '..', '..', 'bin', 'mycha'
 
+
+  installMycha: (testHelperPath, done) ->
+    childProcess = fork @mychaPath, ['install'], silent: yes, cwd: @tmpDir
+    childProcess.stdin.write "#{testHelperPath}\n", 'utf-8'
```

Comment:
This is just blindly filling in the test helper path prompt, right?

What do you think about making `mycha install` accept a command line argument `--testHelperPath` and only prompt the user if it's not provided?